### ISSUE TITLE
function "isFALSE"

### ISSUE THREAD

**Issue by odelaigue** [r-spatial/mapview#177]
With the version on the CRAN, the following example returns an error:

`mapview(breweries)
Error in isFALSE(highlight) : could not find function "isFALSE"`

There is no `isFALSE `function in the mapview package. Is it explorted from another package ?

commit : #46647d1

**Comment by tim-salabim** [comment]
No, `isFALSE` is a base function. But it has only been introduced in R 3.5 and I and the CRAN checks didn't pick this up. The easiest is to upgrade R to 3.5.x and everything should work fine.

**Comment by fdetsch** [comment]
Same here using an older R version w/RStudio Server. If `isFALSE()` has only been introduced in R-3.5, wouldn't this require an entry like this in DESCRIPTION?

```r
Depends: R (>= 3.5)
```

**Comment by tim-salabim** [comment]
Yes it would, but I didn't realise that it was only recently introduced, given that `isTRUE` has been around for a while. And CRAN did not pick this up during submission either. So, for now, we are stuck with this unless we provide some hack around the check where it is used. The reason I use it is that it does not error if the object being tested is e.g. `NULL` which `!` does.

**Comment by fdetsch** [comment]
> The reason I use it is that it does not error if the object being tested is e.g. NULL which ! does.  

Again what learned ;-)

**Comment by odelaigue** [comment]
Thank you very much for your answers!
Indeed, it would be nice to have the test in the DESCRIPTION file.

I will upgrade R from 3.4.2 to R 3.5!

**Comment by edzer** [comment]
Travis checks with R-oldrel break on this, including sf. I'd suggest to use `!isTRUE(...)` instead of `isFALSE`, because of travis and because not everyone has complete control over which version of R they can work with (clusters, windows users without admin rights).

**Comment by tim-salabim** [comment]
@edzer I agree that we need a proper test rather than a hard requirement for R >= 3.5.0. However, in this particular case I need to know whether an argument was set to `FALSE` explicitly (as opposed to `NULL`). Hence `!isTRUE` does not work:
```r
!isTRUE(NULL)
[1] TRUE
!isTRUE(FALSE)
[1] TRUE
```
whereas
```r
isFALSE(NULL)
[1] FALSE
isFALSE(FALSE)
[1] TRUE
```
which is what I need.

**Comment by edzer** [comment]
Yeah, right so you'd need to do `!is.null(x) && !isTRUE(x)`, which IMO weighs against the cost of requiring 3.5.

**Comment by tim-salabim** [comment]
will do so tonight. And resubmit to CRAN after the maintainance break
